### ISSUE TITLE
Pretty Print machines.json

### ISSUE THREAD

**Issue by geoffsimons** [mondaynightpinball/data-archive#55]
Right now the json is serialized to a single line, so any change shows up as a big as chunk of change, instead of individual machine lines. This could also be a case where converting back to csv and using that format going forward would be even better.

**Comment by madcock** [comment]
Whitespace shouldn't matter at all for JSON though. So it should continue to work if you just run it through a validator online that will format it nicely, right? Then in future the diffs would look nice.

**Comment by geoffsimons** [comment]
Correct. I only want whitespace for easier diffs, similar to IPR and
rosters.

On Feb 18, 2019 5:59 PM, "Michael Adcock" <notifications@github.com> wrote:

Whitespace shouldn't matter at all for JSON though. So it should continue
to work if you just run it through a validator online that will format it
nicely, right? Then in future the diffs would look nice.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mondaynightpinball/data-archive/issues/55#issuecomment-464948546>,
or mute the thread
